protos/feast/core/DataSource.proto

```diff
@@ -128,16 +128,14 @@ message DataSource {
   // Java Protobuf class at the given class path
   message KafkaOptions {
     // Comma separated list of Kafka bootstrap servers. Used for feature tables without a defined source host[:port]]
-    string kafka_bootstrap_servers = 1;
+    string bootstrap_servers = 1;
 
     // Kafka topic to collect feature data from.
     string topic = 2;
 
     // Defines the stream data format encoding feature/entity data in Kafka messages.
     StreamFormat message_format = 3;
 
-    // Watermark delay threshold for stream data
-    google.protobuf.Duration watermark_delay_threshold = 4;
```

Comment:
Is there a reason this is being removed?

Reply:
Hello. sorry to come back late.
It seems to have been erased by overwriting my code which is based on previous version. 
I will fix it.